userguide/tutorials/getting_started.adoc

```diff
@@ -136,47 +136,55 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
 
-Users familiar with Java technologies can also install Kill Bill directly in their Web container of choice, such as Tomcat. We recommend using Tomcat version 8.5.
+Users familiar with Java technologies can also install Kill Bill directly in the Web container of their choice, such as Tomcat. We recommend using Tomcat version 8.5.
 
-If you don't have it yet, you can download the Tomcat container http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The documentation is available http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+==== Tomcat Installation
 
-In the rest of this guide, we will assume you have installed Tomcat in a directory called *CATALINA_HOME*.
+. Download the Tomcat container from http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). 
 
-Notes:
+. Install Tomcat as per the instructions specified http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+
+. The rest of the guide assumes that you have installed Tomcat in a directory called *CATALINA_HOME*.
 
-* Unlike other installation methods, you will also need to install and configure your database manually. See https://docs.killbill.io/latest/userguide_deployment.html#_database_engine[our deployment guide] for more details.
-* Installation on Windows might work, but isn't tested nor recommended by the core team.
-* We recommend installing the https://tomcat.apache.org/native-doc/[Apache Tomcat Native Library] (automatically done for you when using Ansible, Docker, or in our Cloud Marketplace instances).
 
 ==== Tomcat configuration
 
-Recommended Tomcat configuration files can be found https://github.com/killbill/killbill-cloud/tree/master/ansible/templates/tomcat/conf[here]. The `.j2` extension is for the Jinja templating language: you must render the files first using your custom variables, which depend on your environment (take a look at an example of such variables https://github.com/killbill/killbill-cloud/blob/master/ansible/group_vars/all.yml[here]). https://ansible.sivel.net/test/[This website] can be used to perform the rendering online if you do not have Jinja installed.
+. Download the Recommended Tomcat configuration files from  https://github.com/killbill/killbill-cloud/tree/master/ansible/templates/tomcat/conf[here]. The configuration files have the `.j2` extension for the Jinja templating language. 
+
+. Render the files  using your custom variables (take a look at an example of such variables https://github.com/killbill/killbill-cloud/blob/master/ansible/group_vars/all.yml[here]). The custom variables depend on your environment.  https://ansible.sivel.net/test/[This website] can be used to perform the rendering online if you do not have Jinja installed.
 
-Once rendered, `setenv2.sh` should be placed in *CATALINA_HOME/bin* and all other files should be placed in *CATALINA_HOME/conf*.
+. Place the `setenv2.sh` in *CATALINA_HOME/bin* 
+
+. Plase the other files in *CATALINA_HOME/conf*.
 
```

Comment:
So basically what I am suggesting is to have a `code section`, where you bullet points are comments, and below the instruction: E.g:

```
...
# Place the other files in *CATALINA_HOME/conf*.
> cp <something> $CATALINA_HOME/conf


```

Reply:
This section is now removed. Also, do you mean add a code section that gives the exact copying instructions?